src/pages/workspace/expensifyCard/WorkspaceCardPageFree.tsx

```diff
@@ -0,0 +1,165 @@
+import {useFocusEffect} from '@react-navigation/native';
+import type {StackScreenProps} from '@react-navigation/stack';
+import React, {useCallback, useMemo} from 'react';
+import type {ListRenderItemInfo} from 'react-native';
+import {FlatList, View} from 'react-native';
+import type {OnyxEntry} from 'react-native-onyx';
+import Button from '@components/Button';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import * as Expensicons from '@components/Icon/Expensicons';
+import * as Illustrations from '@components/Icon/Illustrations';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import {PressableWithoutFeedback} from '@components/Pressable';
+import ScreenWrapper from '@components/ScreenWrapper';
+import useLocalize from '@hooks/useLocalize';
+import useResponsiveLayout from '@hooks/useResponsiveLayout';
+import useThemeStyles from '@hooks/useThemeStyles';
+import localeCompare from '@libs/LocaleCompare';
+import * as PersonalDetailsUtils from '@libs/PersonalDetailsUtils';
+import Navigation from '@navigation/Navigation';
+import type {FullScreenNavigatorParamList} from '@navigation/types';
+import AccessOrNotFoundWrapper from '@pages/workspace/AccessOrNotFoundWrapper';
+import CONST from '@src/CONST';
+import type SCREENS from '@src/SCREENS';
+import type {ExpensifyCard, ExpensifyCardsList} from '@src/types/onyx';
+import WorkspaceCardListHeader from './WorkspaceCardListHeader';
+import WorkspaceCardListRow from './WorkspaceCardListRow';
+
+type WorkspaceCardPageFreeProps = StackScreenProps<FullScreenNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD>;
+
+const stickyHeaderIndices = [0];
+
+// TODO: remove when Onyx data is available
+const mockedCards: OnyxEntry<ExpensifyCardsList> = {
+    test1: {
+        cardholder: {accountID: 1, lastName: 'Smith', firstName: 'Bob', displayName: 'Bob Smith', avatar: ''},
+        name: 'Test 1',
+        limit: '1000',
+        lastFourPAN: '1234',
+    },
+    test2: {
+        cardholder: {accountID: 2, lastName: 'Miller', firstName: 'Alex', displayName: 'Alex Miller', avatar: ''},
+        name: 'Test 2',
+        limit: '2000',
+        lastFourPAN: '1234',
+    },
+    test3: {
+        cardholder: {accountID: 3, lastName: 'Brown', firstName: 'Kevin', displayName: 'Kevin Brown', avatar: ''},
+        name: 'Test 3',
+        limit: '3000',
+        lastFourPAN: '1234',
+    },
+};
+
+function WorkspaceCardPageFree({route}: WorkspaceCardPageFreeProps) {
```

Comment:
Why is this called Free?

Reply:
@mountiny That's the only name I've found in the docs
![image](https://github.com/Expensify/App/assets/23176449/a3e82644-4d68-41d5-a3b5-96d420c911f1)

But maybe **WorkspaceCardPage** or **WorkspaceExpensifyCardPage** would be better

Reply:
I think `WorkspaceExpensifyCardPage` is better, because we're going to have another `Company cards` toggle for third party card workspace feeds, so being specific about this page being the Expensify Card is better.